src/sqlancer/mysql/gen/MySQLDeleteGenerator.java

```diff
@@ -40,17 +43,27 @@ private SQLQueryAdapter generate() {
         // TODO: support partitions
         sb.append(" FROM ");
         sb.append(randomTable.getName());
+        boolean errorsIncluded = false;
```

Comment:
I think the flag and how the logic is implemented is quite confusing. I think we should have a flag `includeExpressionErrors` and set it to `true` in either of the branches where we add expressions.